### PR TITLE
Refactor grouped node details modal

### PR DESCRIPTION
**Pull request by PPiedel**

### CODE REVIEW

File: ui/client/components/graph/node-modal/NodeDetailsModal.js [hunk 1]
```diff
@@ -121,30 +127,6 @@ class NodeDetailsModal extends React.Component {
   }
 
   renderGroup(testResults) {
```
Comment (mproch) [inline ui/client/components/graph/node-modal/NodeDetailsModal.js]: let's remove this method altogether?

File: ui/client/components/graph/node-modal/NodeGroupDetailsContent.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            </div>
+          </div>
+          {node.nodes.map((node, idx) =>
+            <div key={idx}>
+              <NodeDetailsContent isEditMode={false}
+                                  showValidation={true}
+                                  showSwitch={true}
+                                  node={node}
+                                  nodeErrors={props.nodeErrors}
+                                  onChange={() => {
```
Comment (lciolecki) [inline ui/client/components/graph/node-modal/NodeGroupDetailsContent.js]: Empty onChange?

File: ui/client/components/graph/node-modal/NodeDetailsModal.js (hunk 1 above)
Comment (PPiedel) [reply ui/client/components/graph/node-modal/NodeDetailsModal.js]: Done

File: ui/client/components/graph/node-modal/NodeGroupDetailsContent.js (hunk 2 above)
Comment (PPiedel) [reply ui/client/components/graph/node-modal/NodeGroupDetailsContent.js]: Done

**Pull request merged**